python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -0,0 +1,280 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Collection of small images (stamps), each centered on a bright star.
+"""
+
+import collections.abc
+import numpy as np
+
+from lsst import afw
+from lsst import geom
+from lsst.daf.base import PropertySet
+
+
+class BrightStarStamps(collections.abc.Sequence):
+    """Collection of bright star stamps and associated metadata.
+
+    Parameters
+    ----------
+    starStamps : `list` [`dict`]
```

Comment:
Any MutableSequence would do. Changed.

Note I'm not sure if this should be `abc.MutableSequence` in the docstring then, since we do not import `collections.abc`. Is it okay to reference an unimported library in a docstring?

Reply:
I think sphinx does its own imports and look-ups. MutableSequence is a derived class of Sequence, and as you are not using any mutability aspect of the input, it is better to tell people you just need a sequence (i.e. someone could pass a tuple and it would work just fine)